binder/binderlocationbackend_aidl.cpp

```diff
@@ -0,0 +1,1071 @@
+/*
+    Copyright (C) 2015 Jolla Ltd.
+    Copyright (C) 2018 Matti Lehtimäki <matti.lehtimaki@gmail.com>
+    Copyright (C) 2025 Jolla Mobile Ltd.
```

Comment:
```suggestion
    Copyright (C) 2025 Jolla Mobile Ltd
```
In general copyright lines should be written with (c) so lower case c and "Jolla Mobile Ltd" without dot. As repository has (C) here and there better not to deviate here.

Reply:
Done.